build.sbt

```diff
@@ -95,6 +95,7 @@ lazy val commonSettings = Seq(
   fork in Test         := true,
   resolvers            += Resolver.sonatypeRepo("public"),
   resolvers            += Resolver.mavenLocal,
+  resolvers            += "broad-snapshots" at "https://artifactory.broadinstitute.org/artifactory/libs-snapshot/",
```

Comment:
is this to get htsjdk snapshots?  Sucks, but I understand.

Reply:
It is indeed for htsjdk snapshots.